test/unit/application_test.rb

```diff
@@ -230,4 +230,25 @@ class ApplicationTest < ActiveSupport::TestCase
       end
     end
   end
+
+  describe "#latest_deploy_to_each_environment" do
+    should "orders environments" do
+      Deployment.delete_all
+
+      app = FactoryBot.create(:application, name: SecureRandom.hex, repo: "alphagov/#{SecureRandom.hex}")
```

Comment:
Could this be simplified to just `app = FactoryBot.create(:application)` (perhaps by editing the factory method)? Seems a shame to have to specify junk names/repos when we're not asserting anything about them.

Reply:
We have rectified this and removed the unnecessary elements.